app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,39 @@
+class CommentsController < ApplicationController
+
+  def new
+    @comment = Comment.new
+    @post = Post.find(params[:post_id])
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create!(comment_params)
+    redirect_to post_path(@post)
```

Comment:
Note that you don't HAVE to do nested resources. Both posts and comments have unique IDs, so going to `posts/32/comments/24` is redundant: you could just do `/comments/24`. This is implemented here:

https://github.com/ga-dc/tunr_rails_deployment/blob/master/config/routes.rb